magento/pwa-studio
# PWA-2528: Page builder product collection's price+currency is not updating on currency changes

## Description
Original fix from: https://github.com/magento/pwa-studio/pull/3627

While changing currency from the header dropdown, prices in page builder product collections are not getting updated.

To reproduce:

Steps to reproduce the behavior:

1. Go to 'Veni Home page'
2. Change currency from header dropdown.
3. Scroll down to 'Top Sellers' section
4. You can see prices are not updated in chosen currency.

Expected behavior:

Prices should be displayed in user's preferred currency.


## Related Issue







Closes #PWA-2528

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






Test Plan -

- Verify the direct fix for Default store and French store
- Verify direct fix with store code url enabled
- Run existing cypress. suite (Localization test automated)

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
